src/pages/room/[roomId]/[threadId].tsx

```diff
@@ -32,11 +93,34 @@ export const getServerSideProps: GetServerSideProps<ThreadProps> = async (
   const threadId = Number(context.params?.threadId);
   const prisma = new PrismaClient();
   let room = await prisma.room
-    .findFirst({ where: { id: roomId }, include: { users: true } })
+    .findFirst({
+      where: { id: roomId },
+      include: {
+        users: { include: { user: true } },
+        threads: { include: { creator: true } },
+      },
+    })
+    .catch(() => null);
+  let thread = await prisma.thread
+    .findFirst({
+      where: { id: threadId },
+      include: { messages: true },
+    })
     .catch(() => null);
-  if (!room) return { redirect: { destination: "/", permanent: false } };
+  if (!room || !thread)
+    return { redirect: { destination: "/", permanent: false } };
   console.dir(room, { depth: null });
-  return { props: { room, thread: threadId } };
+  const messagesWithUser = await prisma.message.findMany({
+    where: { threadId: threadId },
+    include: { user: true },
+  });
```

Comment:
```suggestion
  });
  prisma.$disconnect();
```
Erstmal, bevor wir ein globales Prisma objekt haben